website/docs/en/plugins/rspack/lightning-css-minimizer-rspack-plugin.mdx

```diff
@@ -62,30 +69,47 @@ At this point, the information that class name b is unused will be obtained via
 .a{color: red}
 ```
 
-### browserslist
-
-- **Type:** `string[]`
-- **Default:** `["defaults"]`
-
-Browserslist targets to compile the CSS for.
-
-### include
-
-- **Type:** `string | RegExp | (string | RegExp)[]`
-- **Default:** `undefined`
-
-Use this to specify which files should be minified.
-
-### exclude
-
-- **Type:** `string | RegExp | (string | RegExp)[]`
-- **Default:** `undefined`
-
-Use this to specify which files should be excluded from minification.
-
-### test
-
-- **Type:** `string | RegExp | (string | RegExp)[]`
-- **Default:** `undefined`
+### minimizerOptions
+
+Configuration passed to Lightning CSS for minification.
+
+Below are the configurations supported. For detailed configuration, please refer to [lightningcss document](https://lightningcss.dev/transpilation.html)
+
+```ts
+type LightningcssFeatureOptions = {
+  nesting?: boolean;
+  notSelectorList?: boolean;
+  dirSelector?: boolean;
+  langSelectorList?: boolean;
+  isSelector?: boolean;
+  textDecorationThicknessPercent?: boolean;
+  mediaIntervalSyntax?: boolean;
+  mediaRangeSyntax?: boolean;
+  customMediaQueries?: boolean;
+  clampFunction?: boolean;
+  colorFunction?: boolean;
+  oklabColors?: boolean;
+  labColors?: boolean;
+  p3Colors?: boolean;
+  hexAlphaColors?: boolean;
+  spaceSeparatedColorNotation?: boolean;
+  fontFamilySystemUi?: boolean;
+  doublePositionGradients?: boolean;
+  vendorPrefixes?: boolean;
+  logicalProperties?: boolean;
+  selectors?: boolean;
+  mediaQueries?: boolean;
+  color?: boolean;
+};
 
-Use this to provide a pattern that css files are matched against. If the filename matches the given pattern, it will be minified, otherwise it won't be.
+type LightningCssMinimizerOptions = {
```

Comment:
We should tell users that all features are excluded by default, and recommend that users to configure `targets`